tests/cyclers/00-daily.t

```diff
@@ -39,8 +39,11 @@ suite_run_ok $TEST_NAME cylc run --reference-test --debug --no-detach $SUITE_NAM
 #-------------------------------------------------------------------------------
 if [[ -f "$TEST_SOURCE_DIR/$TEST_NAME_BASE-find.out" ]]; then
     TEST_NAME="$TEST_NAME_BASE-find"
-    SUITE_DIR="$(cylc get-global-config --print-run-dir)/$SUITE_NAME"
-    (cd "$SUITE_DIR"; find log/job work -type f | sort -V) >"$TEST_NAME"
+    SUITE_RUN_DIR="$(cylc get-global-config --print-run-dir)/$SUITE_NAME"
+    SUITE_WRK_DIR="$(cylc get-global-config -i '[hosts][localhost]work directory')/$SUITE_NAME"
+    (cd "$SUITE_RUN_DIR"; find log/job -type f) >"${TEST_NAME}.tmp"
+    (cd "$SUITE_WRK_DIR"; find work -type f) >>"${TEST_NAME}.tmp"
+    cat ${TEST_NAME}.tmp | sort -V >"$TEST_NAME"
```

Comment:
Can do:

```sh
sort -V "${TEST_NAME}.tmp" >"${TEST_NAME}"
```

Or even avoid using a temporary file with something like this?

```sh
{
    (cd "${SUITE_RUN_DIR}" && find 'log/job' -type f)
    (cd "${SUITE_WRK_DIR}" && find 'work' -type f)
} | sort -V >"${TEST_NAME}"
```

Reply:
A wizardly suggestion!  Done.